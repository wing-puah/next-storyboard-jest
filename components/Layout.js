const Layout = props => {
  const { children } = props;

  return (
    <div>
      { children }
    </div>
  );
};

export default Layout;
