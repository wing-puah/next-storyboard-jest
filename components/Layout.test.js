import { shallow } from 'enzyme';
import Layout from './Layout';

describe('Layout', () => {
  it('renders correct', () => {
    const el = shallow(<Layout />);
    expect(el).toMatchSnapshot();
  });
});
