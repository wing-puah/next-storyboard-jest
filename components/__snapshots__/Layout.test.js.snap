// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correct 1`] = `ShallowWrapper {}`;
